Documentation and change logs for pronouns settings (#19632)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/pronounssettings.md

---
title: "pronounsSettings resource type"

> description: "This resource represents a Setting to control the availability of Pronouns in the tenant"

Reviewer: ```suggestion
description: "Represents a setting to control the availability of pronouns in the tenant"
```

author: "aymen-ms"
ms.localizationpriority: medium
ms.prod: "people"
doc_type: resourcePageType
---

# pronounsSettings resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Represents _settings_ to control the availability of pronouns in the tenant. By default, pronouns are **disabled**. If enabled, users can add and update pronouns in Microsoft 365.

> 

Reviewer 2: Line 16:
I think using "available" or "availability" can be ambiguous, if I understand your feature correctly. (BTW, speaking of which, typically partner teams provide an api.md for the docs for the debut of any API, just so as reviewers, we don't misunderstand how the feature and API behave.)

For example - even if my organization has enabled the display of pronouns, if I don't set pronouns for myself, other users in the organization would not know my pronouns. In other words, my pronouns are still not available because I choose not to or just haven't set them.

It seems to me the setting is about supporting the display of pronouns.

Also, "pronouns in Microsoft 365" is vague and not helpful, especially for a feature like pronouns that impact end user experience - where do pronouns appear in Microsoft 365? 

Can you rephrase the description as:
"Represents the _settings_ to control the display of pronouns in users' profile cards within an organization. By default, pronouns are disabled and not displayed. If enabled, users can optionally add or update their pronouns.

(Separate paragraph) For more information, see Manage display of pronouns using the Microsoft Graph API (and link to your concept topic)."

You can please communicate me offline in case my suggestion about specifying profile cards is incorrect (as we're not supposed to expose future product plans in the public on Github).

Reviewer 2: Per Anton's email on 1/18 that displaying pronouns is one result of toggling on, I'm scoping down to a specific section in the concept topic where I list the exact effects of "enabling" pronouns, and for other places, I try not to tie them to "display" anymore. Instead, I point customers back to the concept topic.

So for line 16, I suggest:
"Represents the settings that manage the support of pronouns in an organization. By default, pronouns are disabled. 

(Separate paragraph) For more information, see Manage pronouns settings for an organization using the Microsoft Graph API (and link to your concept topic)."

> Represents _settings_ to control the availability of Pronouns in the tenant. By default, pronouns are **disabled**. If enabled, users will be able to add and update pronouns in Microsoft 365.

Reviewer: ```suggestion
Represents _settings_ to control the availability of pronouns in the tenant. By default, pronouns are **disabled**. If enabled, users can add and update pronouns in Microsoft 365.
```

Inherits from [entity](../resources/entity.md).

## Methods
|Method|Return type|Description|
|:---|:---|:---|

> |[List pronounsSettings](../api/organizationsettings-list-pronounssettings.md)|[pronounsSettings](../resources/pronounssettings.md)|Get the _settings_ in [pronounsSettings](../resources/pronounssettings.md) object for displaying pronouns in an organization.|

Reviewer: FYI: The api.md file shows "List" as not supported.

Author: Used "List" to make it consistent with other entries in the `orgnaizationsettings`

> |[Update pronounsSettings](../api/pronounssettings-update.md)|[pronounsSettings](../resources/pronounssettings.md)|Update the properties of a [pronounsSettings](../resources/pronounssettings.md) resource to mangage the display of pronouns in an organization.|

Reviewer: ```suggestion
|[Update pronounsSettings](../api/pronounssettings-update.md)|[pronounsSettings](../resources/pronounssettings.md)|Update the properties of a [pronounsSettings](../resources/pronounssettings.md) resource to manage the display of pronouns in an organization.|
```

## Properties
|Property|Type|Description|
|:---|:---|:---|
|isEnabledInOrganization|Boolean| Indicates whether the specified setting is enabled or disabled for the organization. Default value is `false`.|

> 

Reviewer 2: Line 29:
This comment is about the use of the term "specified setting". Technically, just by being a property of the parent pronounsSettings object, there is no extra "specifying" necessary. 
Because at debut for this pronouns API, this property is what the API set is all about, I think developers would find it more helpful if the description is more direct.

Can you rephrase the description as:
"`True` if any set pronouns are displayed in users' profile cards in an organization, `false` otherwise. The default is `false`."

Reviewer 2: Per Anton's email on 1/18..., slight rephrase:
"`True` to enable pronouns settings in the organization, `false` otherwise. The default is `false`, pronouns are disabled."

> |isEnabledInOrganization|Boolean|`true` if the specified settings are enabled for the organization; `false` if the specified settings are disabled for all users. Default is `false`.|

Reviewer: ```suggestion
|isEnabledInOrganization|Boolean| Indicates whether the specified setting is enabled or disabled for the organization. Default value is `false`.|
```

---
title: "pronounsSettings resource type"
description: "This resource represents a Setting to control the availability of Pronouns in the tenant"
author: "aymen-ms"
ms.localizationpriority: medium
ms.prod: "people"
doc_type: resourcePageType
---

# pronounsSettings resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents _settings_ to control the availability of Pronouns in the tenant. Pronouns are *off* by default. When pronouns are enabled, users can add, update, and delete pronouns on their profile card in Microsoft apps and services.

Reviewer 3: not sure if card supports delete per say, also we should not mention probably card here as its one of experices. If we mention it maybe should be "experiences such as profile card"

---
title: "pronounsSettings resource type"

> description: "This resource represents a Setting to control the availability of Pronouns in the tenant"

Reviewer 3: does this have to start with a capital ?

Author: lower case seems to be the norm

Reviewer 3: so will you fix ?

author: "aymen-ms"
ms.localizationpriority: medium
ms.prod: "people"
doc_type: resourcePageType
---

# pronounsSettings resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> 

Reviewer 3: do we have somewhere part about global admin role ?

Author: that is mentioned in the get/patch actions

> Represents a Setting to control the availability of Pronouns in the tenant.

Reviewer 3: needs some extra text about logic
